### DISCUSSION TITLE
I am a very novice coder but I wan't to contribute in some way

### DISCUSSION THREAD

**Discussion by migoa** [a5hk/night-coder#24] [General]
Good day, I am a random person asking if you have any idea on how I can lend a little amount of help.

**Comment by a5hk** [comment]
One way is to create issues (feature requests, enhancements, bug reports, language support)

Contributing to the code is bit more tricky because it is not documented and there is virtually no contribution guide at this moment.
 
I'll try to explain the overall structure of the code a bit here. I use a [theme generator](https://github.com/a5hk/theme-generator) to create this theme and two other themes: [Ice](https://github.com/a5hk/ice) and [Paper](https://github.com/a5hk/paper). It generates ports of these themes for VS Code, vim, bat, and Windows Terminal. Colors are defined [here](https://github.com/a5hk/theme-generator/tree/master/src/common).
For every supported editor, there is a directory with some codes that generate the theme using those colors, [here](https://github.com/a5hk/theme-generator/tree/master/src). For example UI elements of VS Code themes are defined [here](https://github.com/a5hk/theme-generator/blob/master/src/vscode/workbench-common.ts). In the end running `npm run gen` will generate the themes.
